### ISSUE TITLE
Default config.yaml

### ISSUE THREAD

**Issue by jeromerobert** [jm-cc/gcvb#22]
Would it be possible to have this default configuration when the `config.yaml` file is missing ?

```
machine_id: hostname
submit_command: sh
executables: {}
```
with
```python
hostname = platform.node()
```

**Comment by jm-cc** [comment]
I don't think it is possible in the current state. *config.yaml* is used to detect the root of the gcvb repository, allowing commands to work in subfolders.
However, adding a "init" subcommand could generate a *config.yaml* with this kind of default values.

*config.yaml* is also a bad name regarding how the file is used, it will probably be changed to *gcvb.yaml* in a near future.

**Comment by jm-cc** [comment]
Solved with #24  (b43ebf5c6dd5767d19c45d0383fc54a64f25e460)
